MediaBrowser.Common/Json/Converters/JsonNonStringKeyDictionaryConverter.cs

```diff
@@ -0,0 +1,79 @@
+﻿#nullable enable
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Reflection;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace MediaBrowser.Common.Json.Converters
+{
+    /// <summary>
+    /// Converter for Dictionaries without string key.
+    /// TODO This can be removed when System.Text.Json supports Dictionaries without string keys.
```

Comment:
```suggestion
    /// TODO This can be removed when System.Text.Json supports Dictionaries with non-string keys.
```
I think this is more correct